flow/designs/asap7/mock-array-big/macro-placement.tcl

```diff
@@ -1,18 +1,38 @@
 # fish out values from single source of truth: config.py
-set vals [regexp -all -inline {\S+} [exec sh -c {cd designs/asap7/mock-array-big && python3 -c "import config;print(f'{config.rows} {config.cols} {config.pitch_and_margin} {config.die_offset_x} {config.die_offset_y}')"}]]
-lassign $vals rows cols pitch_and_margin die_offset_x die_offset_y
+set vals [regexp -all -inline {\S+} [exec sh -c {cd designs/asap7/mock-array-big && python3 -c "import config;print(f'{config.rows} {config.cols} {config.ce_pitch_x} {config.ce_pitch_y} {config.margin_x} {config.margin_y} {config.placement_grid_x} {config.placement_grid_y}')"}]]
```

Comment:
Bless you for figuring out this nasty bit of code :-)

Ideally I could simply stick to Python across the board, but Python snippets can't be invoked from OpenROAD when OpenROAD is invoked with a .tcl script.